test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OpenApi.yaml

```diff
@@ -680,6 +693,19 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Person'
+          links:
+            Friends:
```

Comment:
Did you run all the test cases? Make sure all the test cases can pass

Reply:
Yes